app/src/main/java/com/example/android/architecture/blueprints/todoapp/statistics/StatisticsViewModel.kt

```diff
@@ -67,6 +69,7 @@ class StatisticsViewModel(
 
         wrapEspressoIdlingResource {
             viewModelScope.launch {
+                // TODO Should this be switched to LiveData observation?
```

Comment:
I thought it was a good idea to show a screen that performed a one-shot operation. But it's probably a good idea to show the reactive pattern everywhere.

Reply:
Yeah, it's the reactive sample - they want to see other patterns, go to another sample.